### DISCUSSION TITLE
HW 3 Part 1

### DISCUSSION THREAD

**Discussion by deena101** [cooperrc/computational-mechanics#54] [Q&A]
I keep getting different errors when I try to load the data, it was working for a second but now its not again. Any thoughts as to why? 
![image](https://user-images.githubusercontent.com/106576120/174501510-ef7d6426-fadc-455c-a9fb-66a252416066.png)

**Comment by FishMacaron** [comment]
I keep getting this one for mine when I input the given code for a :(
> FileNotFoundError: [Errno 2] No such file or directory: 'projectile_coords.npz'

I think a solution for it may be copying and pasting all the info for projectile_coords.npz from part 1's text since there's a projectile_coords.npz in there but I'm not sure since I haven't done it successfully yet.

**Comment by cooperrc** [comment]
If you haven't already, go back to [notebook_01: Projectile Motion](https://cooperrc.github.io/computational-mechanics/module_03/01_Catch_Motion.html#projectile-motion) and follow the steps to collect data from the video and save them as `projectile_coords.npz`. 

You should do three main steps:
1. click on the measured distances to convert pixels -> meters
2. click on the location of the projectile
3. save the time, x-, and y-locations for the projectile ball moving forward
